public/data/harshsinghatz.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Harsh Singh",
+  "bio": "I am an Aspiring full stack developer. I am proficient with JavaScript and looking to do open source contributions frequently",
```

```suggestion
  "bio": "I am an Aspiring full stack developer. I am proficient with JavaScript and looking to do open source contributions frequently!",
```

---

I am unsure why you marked that comment as resolved, it is a nice suggestion!

```suggestion
  "bio": "I am an aspiring full stack developer. I am proficient with JavaScript and looking to do open source contributions frequently!",
```